info-commands.go

```diff
@@ -316,9 +316,11 @@ type TierStats struct {
 
 // KMS contains KMS status information
 type KMS struct {
-	Status  string `json:"status,omitempty"`
-	Encrypt string `json:"encrypt,omitempty"`
-	Decrypt string `json:"decrypt,omitempty"`
+	Status   string `json:"status,omitempty"`
+	Encrypt  string `json:"encrypt,omitempty"`
+	Decrypt  string `json:"decrypt,omitempty"`
```

Comment:
Will those fields contain an error message if it fails to encrypt/decrypt ?

Reply:
@vadmeste yes, that logic remains as old one. Just that all this gets reported at each instance level.
So you feel we should have separate error fields for reporting the encrypt/decrypt errors and make these as booleans?